plugins/modules/notification/mattermost.py

```diff
@@ -55,13 +55,22 @@
   username:
     type: str
     description:
-      - This is the sender of the message (Username Override need to be enabled by mattermost admin, see mattermost doc.
+      - This is the sender of the message.
+      - System Console -> Integrations -> "Enable integrations to override usernames" needs to be set to C(true).
     default: Ansible
   icon_url:
     type: str
     description:
-      - Url for the message sender's icon.
-    default: https://www.ansible.com/favicon.ico
+      - URL for the message sender's icon.
+      - The default is C(https://www.ansible.com/favicon.ico).
```

Comment:
I don't see any code that handles this default. If there is no such code (or `default` is restored), this is a breaking change.